main/src/ca/uwaterloo/flix/Main.scala

```diff
@@ -124,11 +124,16 @@ object Main {
           }
 
         case Command.Init =>
-          Bootstrap.init(cwd)(System.err) match {
-            case Validation.Success(_) =>
+          Bootstrap.init(cwd)(System.err).toResult match {
+            case Result.Ok((_, Nil)) =>
```

Comment:
I think in these matches, I would prefer a non-deep match, i.e. match on Ok and Err. Then in the Ok case further split with an if-then-else on if(errors.isEmpty) ...

Reply:
Scratch that. I change my mind.